### ISSUE TITLE
Upgrade to SDL 2.0.14

### ISSUE THREAD

**Issue by MrStahlfelge** [libgdx/Jamepad#1]
Changelog:

```
2.0.14:
---------------------------------------------------------------------------
General:
* Added support for PS5 DualSense and Xbox Series X controllers to the HIDAPI controller driver
* Added game controller button constants for paddles and new buttons
* Added game controller functions to get additional information:
	* SDL_GameControllerGetSerial()
	* SDL_GameControllerHasAxis()
	* SDL_GameControllerHasButton()
	* SDL_GameControllerGetNumTouchpads()
	* SDL_GameControllerGetNumTouchpadFingers()
	* SDL_GameControllerGetTouchpadFinger()
	* SDL_GameControllerHasSensor()
	* SDL_GameControllerSetSensorEnabled()
	* SDL_GameControllerIsSensorEnabled()
	* SDL_GameControllerGetSensorData()
	* SDL_GameControllerRumbleTriggers()
	* SDL_GameControllerHasLED()
	* SDL_GameControllerSetLED()
* Added the hint SDL_HINT_JOYSTICK_HIDAPI_PS5 to control whether the HIDAPI driver for PS5 controllers should be used.
* Added joystick functions to get additional information:
	* SDL_JoystickGetSerial()
	* SDL_JoystickRumbleTriggers()
	* SDL_JoystickHasLED()
	* SDL_JoystickSetLED()
* Added an API to allow the application to create virtual joysticks:
	* SDL_JoystickAttachVirtual()
	* SDL_JoystickDetachVirtual()
	* SDL_JoystickIsVirtual()
	* SDL_JoystickSetVirtualAxis()
	* SDL_JoystickSetVirtualButton()
	* SDL_JoystickSetVirtualHat()
```

- [X] check new button constants
- [X] check hasLED/setLED
- [X] update game controller db

Implementation on [sdl/2_0_14 branch](https://github.com/libgdx/Jamepad/tree/sdl/2_0_14)

**Comment by MrStahlfelge** [comment]
Tested 85d94f885b9f9b0f6752fde11dbc4922bc0f294d

- [ ] Linux ok
- [ ] Windows no vibration

**Comment by MrStahlfelge** [comment]
Windows problem solved with 718c2fb54e3e2cc6fc171d4af5ae88fca7f7e444

Tests:
- [X] Linux - ok
- [X] Windows - ok
- [ ] Mac
